File: CsoundHelper/src/main/Performer.java
package main;

import csnd6.Csound;
import csnd6.CsoundPerformanceThread;
import csnd6.csnd6;

public class Performer implements Runnable {

	private String orc;
	private String sco;
	private String out;

	public Performer(String orc, String sco, String out) {
		this.orc = orc;
		this.sco = sco;
		this.out = out;
	}

	@Override
	public void run() {
		csnd6.csoundInitialize(csnd6.CSOUNDINIT_NO_ATEXIT | csnd6.CSOUNDINIT_NO_SIGNAL_HANDLER);

		Csound c = new Csound();
		c.SetOption("-o" + out);
		c.CompileOrc(orc);
		c.ReadScore(sco);
		c.Start();

		CsoundPerformanceThread t = new CsoundPerformanceThread(c);
		t.Play();
		t.Join();

		c.Stop();
		c.Cleanup();
	}
}

File: granular_py/performer.py
import ctcsound as csnd6

orc = """
sr = 44100
ksmps = 32
nchnls = 2
0dbfs = 1

instr 1
kcps = 60; frequency
kphs = 0.5; phase
kfmd = 60; frequency variation
kpmd = 0.5; phase variation
kgdur = 0.01; grain duration
kdens = 100; grains per second
imaxovr = 1
kfn = 1; grain waveform
iwfn = 2; window waveform
kfrpow = 1; frequency variation distribution
kprpow = 1; phase variation distribution

ares grain3 kcps, kphs, kfmd, kpmd, kgdur, kdens, imaxovr, kfn, iwfn, \
      kfrpow, kprpow
outs ares, ares
endin
"""

sco = """
f 1 0 0 1 "ah.wav" 0 0 0
f 2 0 512 20 2

i 1 0 1
e
"""

c = csnd6.Csound()
c.setOption("-odac")
c.compileOrc(orc)
c.readScore(sco)

c.start()

while c.performKsmps() == 0:
    pass

c.stop()
